Documentation/build-parameters.rst

```diff
@@ -0,0 +1,98 @@
+Build Parameter Reference
+=========================
+
+**For Developers:** if you add a CMake parameter, make sure to document it here.
+
+For a good overview and easy, we recommend the use of ``ccmake``. To do so, simply replace the use of ``cmake`` by ``ccmake``.
+(i.e. add a "c" in front of the command)
+Alternatively, you may also modify the ``CMakeCache.txt`` in your build directory directly (for it to exist, ``cmake`` needs to be run first).
+
+Equation-specific parameters
+----------------------------
+
+The following parameters will alter the name of the SeisSol executable.
+You may explicitly compile and install multiple of these configurations at the same time—however, you will have to re-do the CMake and build process for each configuration.
+
+- ``CMAKE_BUILD_TYPE``: Either
+
+    * ``Debug``: Also enables assertions. Note that more bugs can appear here than when running with the release options.
+    * ``RelWithDebInfo``: optimizes a bit less than ``Release`` (usually ``-O2``), but offers more debug information.
+    * ``Release``: default value; usually optimizes with ``-O3``.
+- ``EQUATION``: the equation system to compile for
+
+    * ``elastic``: isotropic elastic
+    * ``viscoelastic``: obsolete, use ``viscoelastic2`` instead
+    * ``viscoelastic2``: isotropic viscoelastic. Requires a positive number of relaxation mechanisms (``NUMBER_OF_MECHANISMS``).
+    * ``anisotropic``: anisotropic elastic; essentially uses the same kernels as the ``elastic`` mode, but with more general matrices
+    * ``poroelastic``: isotropic poroelastic
+- ``NUMBER_OF_MECHANISMS``: the number of mechanisms for viscoelastic simulations. For all other equations, this parameter is required to be 0.
+- ``ORDER``: the order of the used polynomial used, that is, its degree plus 1. The order is used for both space and time integration. For example, if you specify order 4, you will be using polynomials of degree 3 in space and time. Note that a higher order can impact the performance greatly.
```

Comment:
All these aren't equation specific parameters, either. Can we push down into Generic parameters or have a separate section for them?